devon4j-migrate/index.asciidoc

```diff
@@ -0,0 +1,38 @@
+= Migrate a devon4j project to the latest version
+====
+The java commandlet allows to setup Java. Also it supports devon4j. The arguments (devon java «args») are explained by the below details: 
+https://devonfw.com/website/pages/docs/devonfw-ide-usage.asciidoc.html#java.asciidoc
+
+This tutorial mainly focuses on the migrate a devon4j project to the latest version.
+## Prerequisites
+* devonfw IDE
+
+## Learning goals.
+Here in this tutorial you will learn 
+* How to migrate an older version of devon4j project to the latest version.
+
+====
+
+[step]
+--
+installDevonfwIde(["java","mvn"], "2020.08.001")
+--
+====
+Now, we will create sample devon4j application with name *sampleapp*. 
+
+[step]
+--
+createDevon4jProject("sampleapp")
+--
+
+Example for migrating a devon4j application. In command promt just type "devon java migrate".
```

Comment:
"Example for migrating a devon4j application." not required..
check the spelling prompt